response_test.go

```diff
@@ -147,3 +147,103 @@ func TestPreloabableResponseAndRemovableResponse(t *testing.T) {
 		}
 	}
 }
+
+// Eligible Response Tests
+
+var eligibleResponseTests = []struct {
```

Comment:
Similar comment to the one in header_test.go: This looks like a duplicate (slightly modified) version of `responseTests`. The existing TestPreloabableResponseAndRemovableResponse can be modified to support EligibleResponse as well by passing in a closure for the function to capture the policy.